paasta_tools/cli/cmds/performance_check.py

```diff
@@ -12,70 +12,66 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-import sys
-
 import requests
+from service_configuration_lib import read_extra_service_information
 
-from paasta_tools.utils import get_user_agent
-from paasta_tools.utils import get_username
-from paasta_tools.utils import load_system_paasta_config
-from paasta_tools.utils import PaastaNotConfiguredError
+from paasta_tools.utils import DEFAULT_SOA_DIR
 from paasta_tools.utils import timeout
 
 
 def add_subparser(subparsers):
     list_parser = subparsers.add_parser(
         'performance-check',
-        description='Performs a performance check (not implemented)',
-        help='Performs a performance check (not implemented)',
+        description='Performs a performance check',
+        help='Performs a performance check',
     )
     list_parser.add_argument(
         '-s', '--service',
         help='Name of service for which you wish to check. Leading "services-", as included in a '
              'Jenkins job name, will be stripped.',
     )
     list_parser.add_argument(
-        '-c', '--commit',
-        help='Git sha of the image to check',
-    )
-    list_parser.add_argument(
-        '-i', '--image',
-        help='Optional docker image to performance check. Must be available on a registry for '
-             'use, like http://docker-dev.yelpcorp.com/example_service-kwa-test1',
+        '-d', '--soa-dir',
+        dest='soa_dir',
+        metavar='SOA_DIR',
+        default=DEFAULT_SOA_DIR,
+        help='Define a different soa config directory',
     )
     list_parser.set_defaults(command=perform_performance_check)
 
 
-def load_performance_check_config():
-    try:
-        return load_system_paasta_config().get_performance_check_config()
-    except PaastaNotConfiguredError as e:
-        print "No performance check config to use. Safely bailing."
-        print e.strerror
-        sys.exit(0)
+def load_performance_check_config(service, soa_dir):
+    performance_check_config = read_extra_service_information(
+        service_name=service,
+        extra_info='performance-check',
+        soa_dir=soa_dir,
+    )
+
+    if not performance_check_config:
+        raise Exception("No performance check config found. Check your performance-check.yaml file.")
```

Comment:
Now, I understand where you are going with this, but because this will be the norm most of the time, I say "don't" raise.

Just let the submit function say:

```
print(No performance-check.yaml. Skipping.)
return
```

And then no need for all the scary messages.